chapter_linear-networks/softmax-regression-scratch_vn.md

```diff
@@ -96,7 +96,11 @@ If we want to keep the number of axes in the original array (resulting in a 2D a
 rather than collapsing out the dimension that we summed over we can specify `keepdims=True` when invoking `sum`.
 -->
 
-*dịch đoạn phía trên*
+Trước khi triển khai mô hình hồi quy softmax, hãy xem xét ngắn gọn cách các toán tử như `sum` hoạt động theo nhiều chiều cụ thể trong một `ndarray`.
+Cho một ma trận `X`, chúng ta có thể tính tổng tất cả các phần tử (mặc định) hoặc chỉ trên các phần tử trong cùng một trục, *ví dụ*, cột (`trục = 0`) hoặc cùng một hàng (`trục = 1`).
+Lưu ý rằng nếu `X` là một mảng có kích thước `(2, 3)`, chúng ta tính tổng các cột (`X.sum (trục = 0`), kết quả sẽ là một vectơ (một chiều) có kích thước là `(3 ,)`.
```

Comment:
```suggestion
Lưu ý rằng nếu `X` là một mảng có kích thước `(2, 3)`, chúng ta tính tổng các cột (`X.sum (axis=0`), kết quả sẽ là một vector (một chiều) có kích thước là `(3 ,)`.
```

Các câu lệnh trong `` không dịch.
vector -> vector.

Reply:
Thank you Tiệp for all the comments!